### PR TITLE
Replace AcousticBrainz favicons with MessyBrainz ones

### PR DESCRIPTION
**Pull request by dpmittal**
https://messybrainz.org/ was displaying favicons of AcousticBrainz instead of favicons of MessyBrainz which needed to be changed

### CODE REVIEW

**Comment by chirlu** [comment]
We [recently decided](https://chatlogs.metabrainz.org/brainzbot/metabrainz/2016-11-21/?msg=3744156&page=2) that favicons should use the no-details version of the logo. (This isn’t implemented everywhere else yet, I know.)

**Comment by dpmittal** [comment]
@chirlu I have made the changes.

**Comment by Freso** [comment]
What was decided was to make AB and MeB act like the other sites, not to always use the "no details" version (even if it turns out to mean the same thing). MB seems to have [a single 16×16 favicon image](https://github.com/metabrainz/musicbrainz-server/tree/master/root), Picard's website [has a single 27×30 one](https://github.com/metabrainz/picard-website/tree/master/website/frontend/static/img) (D:), CritiqueBrainz has [a single 16×16 PNG](https://github.com/metabrainz/critiquebrainz/tree/master/critiquebrainz/frontend/static/images), and ListenBrainz has [a number of favicons](https://github.com/metabrainz/listenbrainz-server/tree/master/webserver/static/img), but practically seems to only use the 32×32 one (the non-32×32 ones are, like for MessyBrainz, actually AcousticBrainz icons).

So maybe what needs to be done is to remove everything except the 32×32 one and only (update and) use that?

**Comment by chirlu** [comment]
Well, the different sizes are for displays of different resolutions. E.g., on my phone with a high-resolution screen, the MB favicon (only 16×16, as you say) looks rather blurry. The actual visible size is in all cases the same (approximately).

In principle, it shouldn’t hurt to have a wider range of sizes except that apparently Firefox will load them all, so that having lots of them takes more disk space/transmission time (32×32 has 0.6 kB, 256×256 has 5 kB or so). [This SO answer](https://stackoverflow.com/a/23734416) recommends, if using PNGs (forget about ICO), using 16×16, 32×32 and “as big as possible”. So we might perhaps drop 64 and 128.

**Comment by chirlu** [comment]
If the selection of sizes is changed, https://github.com/metabrainz/messybrainz-server/blob/master/webserver/templates/base.html#L12-L16 need updating, too. Furthermore (based on the SO answer mentioned above), the 32×32 one should be mentioned last, for the benefit of browsers that ignore `sizes` and only consider the last entry.

**Comment by dpmittal** [comment]
@Freso @chirlu I have removed the icons of 64x64 and 128x128 px and also made the changes in /webserver/templates/base.html

**Review by Freso — APPROVED** [review]

File: webserver/templates/base.html [hunk 1]
```diff
@@ -10,10 +10,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
 
     <link rel="icon" sizes="16x16" href="{{ url_for('static', filename='img/favicon-16.png') }}" type="image/png" />
-    <link rel="icon" sizes="32x32" href="{{ url_for('static', filename='img/favicon-32.png') }}" type="image/png" />
-    <link rel="icon" sizes="64x64" href="{{ url_for('static', filename='img/favicon-64.png') }}" type="image/png" />
-    <link rel="icon" sizes="128x128" href="{{ url_for('static', filename='img/favicon-128.png') }}" type="image/png" />
     <link rel="icon" sizes="256x256" href="{{ url_for('static', filename='img/favicon-256.png') }}" type="image/png" />
+    <link rel="icon" sizes="32x32" href="{{ url_for('static', filename='img/favicon-32.png') }}" type="image/png" />
```
Comment (gentlecat) [inline webserver/templates/base.html:14]: Does the ordering affect anything?

Comment (alastair) [reply webserver/templates/base.html:14]: https://github.com/metabrainz/messybrainz-server/pull/16#issuecomment-272567928

Comment (gentlecat) [reply webserver/templates/base.html:14]: Ah, good to know.

**Pull request merged**